manifest.md

```diff
@@ -54,11 +54,15 @@ Unlike the [Manifest List](manifest-list.md), which contains information about a
         This [descriptor property](descriptor.md#properties) has additional restrictions for `layers[]`.
         Implementations MUST support at least the following media types:
 
-        - [`application/vnd.oci.image.layer.v1.tar+gzip`](layer.md)
-        - [`application/vnd.oci.image.layer.nondistributable.v1.tar+gzip`](layer.md#non-distributable-layers)
+        - [`application/vnd.oci.image.layer.v1.tar`](layer.md)
+        - [`application/vnd.oci.image.layer.v1.tar+gzip`](media-types.md#oci-image-media-types)
+        - [`application/vnd.oci.image.layer.nondistributable.v1.tar`](layer.md#non-distributable-layers)
+        - [`application/vnd.oci.image.layer.nondistributable.v1.tar+gzip`](media-types.md#oci-image-media-types)
```

Comment:
What doesn't look right about it?  We list both the uncompressed and `+gzip` layer types [there][1], and define the `+gzip` form as a gzipped version of the uncompressed layer.  I don't think we need to *define* the `+gzip` forms anywhere else, although there are other cases where it makes sense to list them as “MUST be supported”.

[1]: https://github.com/wking/image-spec/blob/fd9719867cf82ebac6d26bd9212e8bb92a9fd20b/media-types.md#oci-image-media-types